active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+
+If the PreviousButtonStyle or NextButtonStyle sets the Visibility property on the navigational button, it will take precedence over PreviousButtonVisibility or NextButtonVisibility respectively unless they are set to the PipsPagerButtonVisibility value, "Collapsed". 
+If the PreviousButtonVisibility or NextButtonVisibility is set to "Collapsed", the styles set will not overwrite the Visibility of the button itself.
+
+Similarly, the SelectedPipStyle and DefaultPipStyle will take precedence over the values in the theme resources.
+
+![A prototype of five dots, the PipsPager, laid out horizontally with caret-style navigational arrows](images/PipsPager_CustomNavButtons.png)
+
+XAML
+``` XAML
+<Page.Resources>
+    <Style x:Key="NavButtonBaseStyle" TargetType="Button" BasedOn="{StaticResource PipsPagerNavigatonButtonBaseStyle}">
+        <Setter Property="Width" Value="30" />
+        <Setter Property="Height" Value="30" />
+        <Setter Property="FontSize" Value="12" />
+    </Style>
+
+    <Style x:Key="PreviousButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDB;" />
+    </Style>
+
+    <Style x:Key="NextButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDC;" />
+    </Style>
+</Page.Resources>
+
+<muxc:PipsPager x:Name="CustomNavButtonPipsPager"
+  PreviousButtonStyle="{StaticResource PreviousButtonStyle}"
+  NextButtonStyle="{StaticResource NextButtonStyle}"
+  PreviousButtonVisibility="VisibleOnPointerOver" 
+  NextButtonVisibility="VisibleOnPointerOver" />
+```
+## PipsPager member notes
+
+| Name | Description| Default | 
+|:---:|:---|:---|
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1
+| PreviousButtonStyle and NextButtonStyle | Gives the developer the option to customize the style by changing the text or glyph for the edge buttons. | The Previous/Next page button glyph, "Content", and PipsPagerNavigationButtonBaseStyle defined in theme resources.
+| DefaultPipStyle and SelectedPipStyle | Gives the developer the option to customize the style by changing the appearance of the pips. | Based on default and selected pip theme resources and the PipsPagerButtonBaseStyle defined in theme resources.
+| NextButtonVisibility | Sets the visibility of the button that navigates to the next item. | Collapsed |
+| PreviousButtonVisibility | Sets the visibility of the button that navigates to the previous item. | Collapsed |
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
```

Comment:
Design comps disagree. Design comps do not put the selected pip in the middle if there aren't enough pips to fill to the edge. E.g., if you are on the first page, the selected pip is at the far left, rather than putting the selected pip in the center and leaving the space blank on the left.

```suggestion
| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If possible, the UI will scroll the selected pip to the center. | 5
```

Reply:
Recommend: update wording